CRM/Admin/Page/ContactType.php

```diff
@@ -71,6 +71,7 @@ public function browse() {
         'ContactType',
         $value['id']
       );
+      $rows[$key] = array_merge(['class' => ''], $rows[$key]);
```

Comment:
FYI the `+=` operator is more compact (not a blocker)
```suggestion
      $rows[$key] += ['class' => ''];
```